releasenotes-builder/src/main/java/com/github/checkstyle/github/NotesBuilder.java

```diff
@@ -125,6 +129,14 @@ public static Result buildResult(String localRepoPath, String authToken, String
                         remoteRepoPath, issueNo);
                     result.addError(error);
                 }
+                if (issueLabel.moreThanOneLabel()) {
```

Comment:
you can not add new method in jdk String object.
you need to make method `moreThanOneReleaseLabel(issueLabels)`

```
 private static String getIssueLabelFrom(GHIssue issue) throws IOException {
        final Collection<GHLabel> issueLabels = issue.getLabels();
        final Optional<GHLabel> label = issueLabels.stream()
            .filter(input -> Arrays.binarySearch(Constants.ISSUE_LABELS, input.getName()) >= 0)
            .findFirst();   // !!!!!!!!!!! remove this line, and return collection
```

Reply:
Right, I'll do the changes. 

for method, moreThanOneReleaseLabel
Should I make a condition in method if it has more than one label?

Reply:
yes